.github/workflows/deploy.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
     steps:
       - id: setup
         name: Setup
-        uses: scientist-softserv/actions/setup-env@main
+        uses: scientist-softserv/actions/setup-env@arm-wrestle
```

Comment:
why the deploy changes?

Reply:
Residual from testing. Removing shortly.